lib/python/test/KHI_growthRate/MainTest.py

```diff
@@ -0,0 +1,38 @@
+import sys
+from time import *
+
+lt = localtime()
+date = strftime("date: %d.%m.%Y", lt)
+timeOfDay = strftime("time: %H:%M:%S", lt)
+
+try:
+	sys.path.append(sys.argv[1])
+	from TestKHI import TestKHI
+	from ViewerKHI import ViewerKHI
+
+	field = ["total", "B_x", "B_y", "B_z", "E_x", "E_y", "E_z"]
+
+	paramDir = sys.argv[2]
```

Comment:
Where should param dir points too? From the `tpl` it looks like it is the folder of `MainTest.py` which is equal to the dirname of the first parameter.

Reply:
not really, paramDir points to the folder containing the .param files. sys.argv[0] is MainTest.py directly, [1] is the folder in which MainTest.py is located (I'm still checking your previous comment), [2] is as described above and [3] points to simOutput